layer/nrlf/nrlf/core/authoriser.py

```diff
@@ -74,11 +73,25 @@ def parse_headers(
     _raw_connection_metadata = _headers.get(CONNECTION_METADATA, "{}")
     try:
         connection_metadata = ConnectionMetadata.parse_raw(_raw_connection_metadata)
+    except ValidationError:
+        return PipelineData(
+            error={
```

Comment:
seems like we never need the intermediate dict `message`, why not just `"error": "blah"`